README.md

```diff
@@ -1 +1,57 @@
-# se452-group7-project
\ No newline at end of file
+# se452-group7-project
+
+This repository is the group project for SE453 - Group 7 - Fall 2024   
```

Comment:
Can you please check the sentence formation here.....it can be like 'This repository is for ...... or This repository serves as ....